GitUI/CommandsDialogs/RepoHosting/ViewPullRequestsForm.cs

```diff
@@ -330,6 +333,8 @@ private void LoadDiffPatch()
         }
 
         private Dictionary<string, string> _diffCache;
+        private bool _cloneWithHttps;
```

Comment:
Nitpick: Define the private fields in one place and sort them alphabetically, please; also applies to the definitions at line 59-61.

Reply:
`_diffCache`, too, please.